src/amber/controller/base.cr

```diff
@@ -32,5 +32,9 @@ module Amber::Controller
     def session
       context.session
     end
+
+    def halt(status_code = 200, output = "")
+      raise Amber::Exceptions::HaltRequest.new(status_code, output)
```

Comment:
Yes, I reverted back something was not working and was wondering why I removing this commit from the PR. Hence the WIP